src/components/tutorials/advanced-tutorial.js

```diff
@@ -16,24 +16,23 @@ class AdvancedTutorial extends React.Component {
-            <h2 className='ds-h2' id='submitting-with-performance-data'>
+            <h2 className='ds-h2' id='add-more-measures'>
               <a
                 className='tutorial-header-link'
-                href='#submitting-with-performance-data'>
-              Creating a submission with embedded performance data
+                href='#add-more-measures'>
+              Add more measures to an existing measurement set
             </a>
             </h2>
-            <p>Previously, we created a submission and added a measurement set in two requests. It's common to want to do these at the same time - the first time we want to submit performance data for any individual, for instance. It's convenient to be able to do both, so let's try that with ACI performance data.</p>
-            <p>Here's a <code>POST</code> request to create a submission. You can pass an <code>Accept</code> header to specify the API version and the desired response format by using our custom mime type, <code>application/vnd.qpp.cms.gov.v1+json</code>. We support JSON and XML by using <code>+json</code> or <code>+xml</code>. Right now, only version <code>v1</code> is supported. You can also continue to use a standard <code>application/json</code> or <code>application/xml</code> header, which will point to the latest version (currently v1).</p>
-            <p>We're including the  measurement set and submission together by nesting the measurement set inside the submission info, much like we nested measurements within measurement sets.</p>
+            <p>Previously, we created a measurement set containing quality measures. As you may know, a clinician must submit 6 Quality measures in order to get a score for that category. So let's try adding 2 more measures to the measurement set we created in the previous tutorial.</p>
+            <p>Here's a <code>PATCH</code> request to add more measures to an existing measurement set. You can pass an <code>Accept</code> header to specify the API version and the desired response format by using our custom mime type, <code>application/vnd.qpp.cms.gov.v1+json</code>. We support JSON and XML by using <code>+json</code> or <code>+xml</code>. Right now, only version <code>v1</code> is supported. You can also continue to use a standard <code>application/json</code> or <code>application/xml</code> header, which will point to the latest version (currently v1).</p>
             <p>The request payload on the right can be harder to read, but it contains the same info as below formatted exactly as it's sent through the API. Check it out then click response!</p>
             <InlineApiExample
-              verb='POST'
-              url='/submissions'
+              verb='PATCH'
+              url='/measurement-sets'
               params={
```

Comment:
update the response payload to:


```

{
  "error": {
    "type": "ValidationError",
    "message": "invalid measurement-set object",
    "details": [
      {
        "message": "field 'measureId' in MeasurementSet.measurements[0] is invalid: IA_EPA_10 does not exist, see qpp-measures-data for list of valid measureIds",
        "path": "$.measurements[0].measureId"
      }
    ]
  }
}
```

Reply:
add measurementSetId as a param. the measurementSetId is 0313d351-a094-4dc2-b632-23cd88bffb5e